src/Processor/Backtrace.php

```diff
@@ -59,6 +59,31 @@ public function process(array $event)
         return $event;
     }
 
+    /**
+     * Add a custom namespace to be ignored from the backtrace
+     *
+     * @param string $namespace
+     * @return self
+     */
+    public function addIgnoredNamespace($namespace)
```

Comment:
Can it be changed to constructor injection?
I see no reason to change value after processor was created, as new namespaces to ignore should really be added by adding more configured processors